examples/java/psg-spring-boot-demo/psg-spring-boot-demo.iml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Isn't it idea specific file? Should it be committed?

Reply:
removed